Meadowlark-js/backends/meadowlark-mongodb-backend/src/repository/Db.ts

```diff
@@ -33,6 +33,7 @@ export async function getNewClient(): Promise<MongoClient> {
 
     // Note this does nothing if the index already exists (triggers an index build otherwise)
     await collection.createIndex({ outRefs: 1 });
+    await collection.createIndex({ existenceIds: 1 });
```

Comment:
index for existenceIds

Reply:
This caused me to ask: what are the limits on indexes? [We're in safe territory](https://www.mongodb.com/docs/manual/reference/limits/#indexes), not surprisingly.